### ISSUE TITLE
constant Disk activity on synology Tdarr docker 

### ISSUE THREAD

**Issue by michpas** [HaveAGitGat/Tdarr#284]
**Describe the bug**
Running Tdarr creates every 1-2 seconds Disk IO activities.
Even when all libraries are completly turned off.

**To Reproduce**
Install Tdarr docker on synology
Turn on disk monitoring.
Wait a while to stabailize/startup
See 10% constant disk activity. 
Turn off docker.
Disk activity will immediately drop.

**Expected behavior**
I expected without any libraries active (eg. no folder watches) it wouldn't generate constant disk activity

**Screenshots**
![image](https://user-images.githubusercontent.com/66222810/94374740-9e2f4500-010e-11eb-9d46-f65b43a6f63d.png)

**Desktop (please complete the following information):**
Synology.
Docker. Latest public.

**Additional context**
Also I notice a very constant sound on the harddisk activity. Maybe 1-2 seconds. Maybe this harddisk activity is due to the database polling (I think that one is set to evey second)

**Comment by HaveAGitGat** [comment]
Hi, this should be fixed in the 1.3001 update which is currently in the pro_nightly container, just getting some feedback on it.
```
Performance Update
- Beta v1.3000 release [27th September 2020]:
- Massively increased performance across application (10* faster queue builder, 10* less DB usage, up to 200* faster file checking on startup, ~20% less RAM, far more responsive UI especially when scanning files)
- Fixed UI bug when creating a backup (backup size showing much lower than it should be)
- Fixed folder watch bug (error saying txt file doesn't exist)
- Moved scheduled hourly 'Scan (Find new)' to its own option instead of being enabled when 'Scan on start' enabled.
- Beta v1.3001 release [1st October 2020]:
- Massively reduced system load when running 'Scan (Find New)' or when folder watcher adding lots of files (~150* faster scanner prep time with 100,000 file library)
- Fixed DB initialisation memory bug on startup if 30,000+ files
```

**Comment by HaveAGitGat** [comment]
Please use the latest Docker tag (Tdarr v1.3003), which has the performance enhancements in. Re-open if needed.
